nasa_wildfires/__init__.py

```diff
@@ -1,58 +1,51 @@
 import csv
 import geojson
 import requests
+from .area import regions
 
+areas = list(regions.keys())
```

Comment:
Since we are using the term "region" in the public API for users, let's keep that lingo going here. And since it's in the global namespace the Python tradition is to make the variable all caps. How about `REGION LIST`?